book/invalidation.md

```diff
@@ -1146,7 +1146,7 @@ build the list of children, and then `set` it as the new value of the
 
 [^perhaps-local]: Perhaps the nicest design would thread a local
 `children` variable through all of the methods involved in line
-layout, similar to how we handle `paint`.
+layout, similar to `tree_to_list`.
```

Comment:
This is important—`paint` no longer threads an output array through a recursive function call.